.changeset/rude-goats-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/eslint-plugin': patch
+---
+
+bump eslint dependencies
```

Comment:
This is a `major` breaking change thanks to the removal of the the `jest/no-jest-import` rule. Please update the update type.


Please write a more complete changelog entry. Please detail what packages were bumped, and to what version, such as https://github.com/Shopify/web-configs/blob/main/packages/eslint-plugin/CHANGELOG.md#breaking-change. You should also detail any potentially breaking changes - notable the removal of the `jest/no-jest-import` rule.